libmscore/layout.cpp

```diff
@@ -296,7 +296,7 @@ void Score::layoutChords1(Segment* segment, int staffIdx)
                               else
                                     break;
                               }
-                        for (int i = int(downStemNotes.size()) - 1; i >= 0; --i) {
+                        for (size_t i = 0; i < downStemNotes.size(); ++i) {
```

Comment:
This does change the loop's direction, but I don't see any reason why upstem notes are processed left to right, but downstem notes should be processed right to left. And mtests seem to agree with me.